go.mod

```diff
@@ -40,15 +40,20 @@ require (
 	github.com/cloudflare/circl v1.3.7 // indirect
 	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc // indirect
 	github.com/fatih/color v1.16.0 // indirect
+	github.com/felixge/httpsnoop v1.0.4 // indirect
```

Comment:
why do you have go file changes? can you drop all of these?

Reply:
Sure, lemme drop the go.mod change, I was assuming gomod and gosum are committed in this repo